plugins/woocommerce-blocks/assets/js/blocks/mini-cart-interactivity/style.scss

```diff
@@ -0,0 +1,225 @@
+.wc-block-mini-cart-interactivity {
```

Comment:
copy pasted, added `-interactivity` in a couple places. This would all get refactored again later when we move to replacing the original block.